contracts/drafts/meta-tx/SigningRecipient.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.5.0;
+
+import "./GSNRecipient.sol";
+import "../../cryptography/ECDSA.sol";
+
+contract SigningRecipient is GSNRecipient {
```

Comment:
I don't like this name because it's not the recipient who signs. Don't have a better name atm.